main.go

```diff
@@ -683,3 +692,39 @@ func (cmd *goCmd) run() error {
 
 	return nil
 }
+
+type goEnv struct {
+	ReleaseTags map[string]bool
+	GOFLAGS     string
+	GOPATH      string
+	GOPROXY     string
+	GOBIN       string
+}
+
+func getGoEnv() (goEnv, error) {
+	var res goEnv
+	{
+		cmd := exec.Command("go", "env", "-json")
+		out, err := cmd.CombinedOutput()
+		if err != nil {
+			return goEnv{}, fmt.Errorf("failed to get go env: %v\n%s", err, out)
+		}
+		if err := json.Unmarshal(out, &res); err != nil {
+			return goEnv{}, fmt.Errorf("failed to unmarshal go env: %v (output was %q)", err, out)
+		}
+	}
+	{
+		cmd := exec.Command("go", "list", "-f={{context.ReleaseTags}}", "runtime")
```

Comment:
cmd := exec.Command("go", "list", `-f={{join "\n" context.ReleaseTags}}`, "runtime")

?
Then you could just use `strings.Fields` below.

Reply:
Thank you. I _keep_ forgetting about `join`!